readme.md

```diff
@@ -22,6 +22,8 @@ This list is mainly about [CSS](https://developer.mozilla.org/docs/Web/CSS) –
 
 ## CSS References
 
+- [Learn CSS - Web.dev](https://web.dev/learn/css/) - An evergreen CSS course and reference to level up your web styling expertise.
```

Comment:
Please remove this. I don’t think we need another reference in this awesome list.